### ISSUE TITLE
cars_attributes ver 3.04 does not work in Osclass <3.2

### ISSUE THREAD

**Issue by conejoninja** [osclass/plugin-cars_attributes#1]
It appears that config.php was left out for Osclass <3.2

Original issue : https://github.com/osclass/osclass-plugins/issues/163

**Comment by alexgreece** [comment]
Yep plugin doesn't work on 3.2 no matter what you search it doesn't bring any results please fix it :(
